.changeset/README.md

```diff
@@ -0,0 +1,8 @@
+# Changesets
```

Comment:
This feels unrelated to adding support for `eth_getTransactionCount`

Reply:
This is auto-generated initialization files by versioning manager we use. I forgot to add it in the corresponding pr